src/utils/customFetch.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpVerbs } from "src/@types/HttpVerbs";
+
+type Body = {
+  [key: string]: string;
+};
```

Comment:
As discussed, if you use the global Body type, you can remove this.

Reply:
OK I'm following your advices, I remove those lines and use type Record instead